src/richText.js

```diff
@@ -213,55 +216,130 @@ define([
     }
 
     /**
-     * Takes a value from xml and replaces xpaths with bubbles to display in the
-     * editor
+     * Replace <output> tags with bubble markup
      *
-     * This does not add any <p> or <br> tags, only newlines as used in xml
+     * @param withClose - Create bubbles with close buttons if true.
+     * @param escape - If true, escape HTML except for bubble markup.
+     */
+    function bubbleOutputs(text, form, withClose, escape) {
+        var el = $('<div>').html(text),
```

Comment:
text should be escaped like https://github.com/dimagi/Vellum/pull/554/files#diff-85ed993f6f09d78b2af63dbea82386a3R333

![selection_001](https://cloud.githubusercontent.com/assets/1471773/10857052/26997fba-7f22-11e5-8fa8-9e7a29276477.png)